content/applications/sales/subscriptions/payment_providers/sdd.rst

```diff
@@ -0,0 +1,216 @@
+   :alt: The SEPA Direct Debit Payment Provider form.
+
+Once all necessary configuration settings are complete, the :icon:`fa-eye-slash`
+:guilabel:`Unpublished` smart button located at the top of the page automatically changes to
+:icon:`fa-globe` :guilabel:`Published` to show the |sdd| payment option is now visible to customers.
+Only keep the payment method published if payments are ready to be received.
+
+.. tip::
+   It is possible to keep the payment method enabled, but **not** visible to customers, until
+   payments are ready to be received. To do this, set the smart button to :icon:`fa-eye-slash`
+   :guilabel:`Unpublished` after setting the :guilabel:`State` to :guilabel:`Enabled`.
+
+.. note::
+   The |sdd| configuration menu can also be accessed by navigating to :menuselection:`Settings app
+   --> Website`, and in the :guilabel:`Shop - Payment` section, click the :guilabel:`Configure SEPA
+   Direct Debit` button.
+
+Test mode
+~~~~~~~~~
+
+The |sdd| payment method can be tested before it is accessed by customers. First, ensure the test
+creditor identifier is :ref:`configured <subscriptions/sepa-sdd-configuration>` in **Settings**
+before continuing.
+
+.. warning::
+   It is recommended to use the :guilabel:`Test Mode` setting on a duplicate (or test) database to
+   avoid potential issues with invoice numbering.
+
+Next, select the :guilabel:`Test Mode` option in the :guilabel:`State` field on the form. Upon
+selecting :guilabel:`Test Mode`, an orange :guilabel:`Test mode` banner appears in the upper-right
+corner of the form.
+
+.. note::
+   By default, the payment provider remains :guilabel:`Unpublished` in :guilabel:`Test Mode`, so it
+   is not visible to visitors.
+
+Navigate to :menuselection:`Sales app --> Orders --> Quotations` and click the :guilabel:`New`
+button to create a new sales order. Select a :guilabel:`Customer`. Select any pay period in the
+drop-down menu next to :guilabel:`Recurring Plan` and select the pricelist using EUR in the
+drop-down menu next to :guilabel:`Pricelist`. Finally, select a subscription product in the
+:guilabel:`Order Lines` tab of the sales order.
+
+Next, confirm the sales order by clicking the :guilabel:`Confirm` button, then click the
+:guilabel:`Preview` button to open a preview of the customer invoice. Scroll down to the
+:guilabel:`Anticipate payment` section to find the SEPA Direct Debit payment method.
+
+.. tip::
+   It can be confirmed that test mode is still enabled if the :icon:`fa-eye-slash` (unpublished) and
+   :icon:`fa-exclamation-triangle` (test mode) icons are present next to :guilabel:`SEPA Direct
+   Debit`.
+
+   .. image:: sdd/sdd-anticipated.png
+      :align: center
+      :alt: The view of the SEPA Direct Debit payment method presented to customers at checkout.
+
+Next, enter a test IBAN number, such as `GB33BUKB20201555555555`, and click :guilabel:`Pay`. Return
+to the sales order in the **Sales** app and confirm the :guilabel:`Pending transaction` tag is
```

Comment:
```suggestion
to the sales order in the **Sales** app, and confirm the :guilabel:`In Progress` tag is
```
I'm seeing that as 'In Progress' -- might be worth a quick double-check during your revisions...